arch/arm64/src/imx9/imx9_lpspi.c

```diff
@@ -1338,6 +1341,17 @@ static void imx9_lpspi_exchange(struct spi_dev_s *dev,
       return;
     }
 
+  /* Check if the transfer is too long */
+
+  if (nbytes > CONFIG_IMX9_LPSPI_DMA_BUFFER_SIZE)
+    {
+      /* Transfer is too long, revert to slow non-DMA method */
+
+      DEBUGPANIC();
```

Comment:
why panic?

Reply:
It's just a debugpanic, so the developer notices that the DMA buffer size is too small. It has to be a run-time check. The non-DMA method is slow because it's just a polling / busylooping method.

Reply:
so, should we change to warning?

Reply:
Sure, change to spiwarn() trace or something else ?

Reply:
ok.